nipype/interfaces/freesurfer/base.py

```diff
@@ -162,8 +162,45 @@ def version(self):
                 return ver.rstrip().split('-v')[-1]
 
 
+class FSSurfaceCommand(FSCommand):
+    """Support for FreeSurfer surface-related functions.
+    For some functions, if the output file is not specified starting with
+    'lh.' or 'rh.', FreeSurfer prepends the prefix from the input file to the
+    output filename. Output out_file must be adjusted to accommodate this.
+    By including the full path in the filename, we can also avoid this behavior.
+    """
+    def __init__(self, **inputs):
+        super(FSSurfaceCommand, self).__init__(**inputs)
+
+    def _get_filecopy_info(self):
+        """Filename normalization routine to perform only when run in Node
+        context
+        """
+        self._normalize_filenames()
+        return super(FSSurfaceCommand, self)._get_filecopy_info()
+
+    def _normalize_filenames(self):
+        pass
+
+    @staticmethod
+    def _associated_file(in_file, out_name):
+        """Based on MRIsBuildFileName in freesurfer/utils/mrisurf.c
+
+        Use in_file prefix to indicate hemisphere for out_name, rather than
+        inspecting the surface data structure.
+        """
+        path, base = os.path.split(out_name)
+        if path == '':
+            _, in_file = os.path.split(in_file)
```

Comment:
This should be `path, in_file = ...`. The logic here is "if `out_file` doesn't have path information, use `in_file`'s path".